packages/docs/src/pages/startup/examples.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Examples
+weight: 4
+---
+
+<h2 id="examples" class="ds-h2 ds-u-color--primary-darker">Examples</h2>
+
+Additional examples of the design system in use can be viewed on GitHub. These projects demonstrate the various ways you can incorporate the design system into your development process and various use cases.
+
+<a href="https://github.com/CMSgov/design-system/tree/master/examples/" class="ds-c-button">Browse example projects</a>
+
+<h2 id="need-help" class="ds-h2 ds-u-color--primary-darker">Need help or ran into an issue?</h2>
```

Comment:
Thoughts on having this same Need Help heading on every page of the startup? I think it definitely makes sense on the first section (Installation), but I'm not sure it needs to be on every page. I don't know what a best practice is here, though.

Reply:
I'd like to keep it for now -- I'm hearing some feedback that finding help is a little tough, and there's no harm in having it repeated here.